pkg/internal/mock/provider/mock_provider.go

```diff
@@ -30,6 +30,10 @@ type Provider struct {
 
 // Service return service
 func (p *Provider) Service(id string) (interface{}, error) {
+	if p.ServiceMap[id] != nil {
```

Comment:
I guess `p.ServiceErr` should take precedence.

Reply:
[Done](https://github.com/hyperledger/aries-framework-go/compare/3833ed3479ac417972cff8aad5a07230c2f6f54b..9dcd5f3aff2537b1907bafee711df2921a81201e#diff-2f0c4d1de918ed4abf739e5784fa0a64R33).